src/master-playlist-controller.js

```diff
@@ -233,6 +234,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     this.seekable_ = videojs.createTimeRanges();
     this.hasPlayed_ = () => false;
+    this.playlistEnd = playlistEnd;
```

Comment:
I attached this function to ```MasterPlaylistController``` because I was trying to call this function in the ```videojs-contrib-hls.test.js``` file. but maybe there is better way to do this